cloudslang-runtime/src/main/resources/scripts/get_sp_var.py

```diff
@@ -0,0 +1,8 @@
+def get_sp_var(key):
+  try:
+    value1 = get_from_smaller_context(key)
+  except NameError:
+    value1 = globals().get(key)
+  accessed(value1)
+  property_value = sys_prop.get(value1)
+  return property_value
```

Comment:
you might want to have a new line at the end of the file

Reply:
Done